HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -1016,6 +1016,8 @@ def initialize(name, used_by_unavailable_periods, can_be_stochastic, type)
   # periods CSV (e.g., heating), and/or C) EnergyPlus-specific schedules (e.g., battery_charging).
   Columns = {
     Occupants: Column.new('occupants', true, true, :frac),
+    EVOccupant: Column.new('ev_occupant', true, true, :int),
+    PresentOccupants: Column.new('present_occupants', true, true, :int),
```

Comment:
Why do we need these two new columns? As far as I can tell they aren't used, and they seem confusing.

Reply:
EVOccupant is the occupancy schedule of the occupant that is driving the EV. 
PresentOccupants is the occupancy of all occupants.
EVOccupant column can be used, for example, to simulate demand response for EV where EV charging is delayed. Without knowing the EV occupant schedule it won't be possible to align charging with the time the occupant is home.
PresentOccupants is mostly for debugging/postprocessing purpose. I have used this to generate graphics like this below where I am looking at the away hours of EV occupants compared to away hours of all occupants.
![image](https://github.com/user-attachments/assets/ecd5d271-6d05-4548-b9a0-ebd3640c4296)

Reply:
I see. Thanks for the description. What it cause any issues for your use case if we only exported these columns when the `debug` flag is used?

Reply:
Yeah, I think we can switch to using debug flag for outputting these. Or, can this be controlled through output variables?